src/database/model/user.model.js

```diff
@@ -28,18 +28,18 @@ const userSchema = new mongoose.Schema({
     required: [true, 'Please provide a password'],
     minlength: 8,
     select: false
-  },
-  passwordConfirm: {
-    type: String,
-    required: [true, 'Please confirm your password'],
-    validate: {
-      // This only works on CREATE and SAVE!!!
-      validator: function(el) {
-        return el === this.password;
-      },
-      message: 'Passwords are not the same!'
-    }
-  }
+   },
+  // passwordConfirm: {
+  //   type: String,
+  //   required: [true, 'Please confirm your password'],
+  //   validate: {
+  //     // This only works on CREATE and SAVE!!!
+  //     validator: function(el) {
+  //       return el === this.password;
+  //     },
+  //     message: 'Passwords are not the same!'
+  //   }
+  // }
```

Comment:
Don't forget, commented source codes are not pushed for the best practice and code quality.